e2e/testdata/low-priority-pause-deployment.yaml

```diff
@@ -23,6 +23,7 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: pause-deployment
+  namespace: default
   labels:
```

Comment:
Might want to add the test labels on here so normal cleanup catches it. i don't think MustKubectl applies them the same way KubeClient.Create does. Or switch to using KubeClient.

Reply:
Added the test label